src/modules/TTS/splitting.js

```diff
@@ -245,17 +245,10 @@
       /** @type {string} */
       var wholeText = next.wholeText || "";
       var contentsTrimmed = wholeText.trim();
-      // var contents = wholeText;
 
       // If there's no text left after trimming whitespace, continue the loop
       if (contentsTrimmed.length) {
-        // insert leading space if there was one
-        allElements.push(
-          createElement(F, "whitespace", " ", preserveWhitespace)
-        );
-        if (wholeText[0] === " ") {
-          allElements.push(createText(" "));
-        }
+        allElements.push(createText(" "));
```

Comment:
This is adding spaces where ones previously did not exist.   Consider using previous whitespace check before pushing space:
```
// insert leading space if there was one
if (wholeText[0] === " ") {
    allElements.push(createText(" "));
}
```

Reply:
reverted this